### PR TITLE
created contact page

### LINKED ISSUE #25: Creating Contact Us Page

### PR DESCRIPTION
**Pull request by musavveer**
Please Provide The following Details About your PR

Type x Inside the checkbox to tick it

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #25 

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

I have attached the screenshot.
Screenshot 2022-10-24 at 8 52 12 PM

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: templates/contact.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+</head>
+
+<body>
+
+  <h1>Contact Form</h1>
+
+  <div class="container">
+    <form action="/action_page.php">
+      <label for="fname">Name</label>
+      <input type="text" id="fname" name="firstname" placeholder="Full Name..">
```
Comment (kailashchoudhary11) [inline templates/contact.html]:
```suggestion
      <input type="text" id="name" name="name" placeholder="Full Name..">
```

File: templates/contact.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  </style>
+</head>
+
+<body>
+
+  <h1>Contact Form</h1>
+
+  <div class="container">
+    <form action="/action_page.php">
+      <label for="fname">Name</label>
```
Comment (kailashchoudhary11) [inline templates/contact.html]:
```suggestion
      <label for="name">Name</label>
```

File: templates/contact.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    }
+  </style>
+</head>
+
+<body>
+
+  <h1>Contact Form</h1>
+
+  <div class="container">
+    <form action="/action_page.php">
```
Comment (kailashchoudhary11) [inline templates/contact.html]:
```suggestion
    <form method="post">
```

File: templates/contact.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      <label for="fname">Name</label>
+      <input type="text" id="fname" name="firstname" placeholder="Full Name..">
+
+      <label for="lname">Email</label>
```
Comment (kailashchoudhary11) [inline templates/contact.html]:
```suggestion
      <label for="email">Email</label>
```

File: templates/contact.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      <input type="text" id="lname" name="email" placeholder="Email">
```
Comment (kailashchoudhary11) [inline templates/contact.html]:
```suggestion
      <input type="text" id="email" name="email" placeholder="Email">
```

**Review by kailashchoudhary11 — CHANGES_REQUESTED** [review]

**Comment by kailashchoudhary11** [comment]
@musavveer try to use classes and ids for css instead of targeting elements!

**Review by kailashchoudhary11 — APPROVED** [review]
Looks Good to me

**Comment by kailashchoudhary11** [comment]
@musavveer  Don't Forget to Star the Repository!

**Comment by musavveer** [comment]
> @musavveer Don't Forget to Star the Repository!

Do include hacktoberfest accepted label .>>>

**Comment by kailashchoudhary11** [comment]
> > @musavveer Don't Forget to Star the Repository!
> 
> Do include hacktoberfest accepted label .>>>

it is not required all the PRs to repo are included for hacktoberfest

**Pull request merged**